assets/compat/dependencies.yaml

```diff
@@ -1,6 +1,11 @@
+# sets either lnd or c-lightning as required dependency if selected
 lnd:
   condition: '''lightning.type = "lnd"'
   health_checks: []
 c-lightning:
   condition: '''lightning.type = "c-lightning"'
+  health_checks: []
+# sets internal bitcoin node or internal bitcoin proxy as required dependency if selected
+bitcoin:
```

Comment:
shouldn't this say `bitcoind`?

Reply:
No, it's not a reference to a package-id here, it's a key defined however you want.

Reply:
oh im dumb, when i was looking at this on mobile, i thought it was the config_spec.yaml file. you are totally correct, it should be bitcoind